src/utils/translation.ts

```diff
@@ -16,6 +16,7 @@ import gu from "./translations/gu";
 import vi from "./translations/vi";
 
 import { storage } from "../data/Storage";
+import ta from "./translations/ta";
```

Comment:
Please import your localization file above like we doing this with other localization files

Reply:
Do you mean to import the files in the correct order, similar to how the others are organized?